.github/workflows/check-requiredment.yml

```diff
@@ -0,0 +1,23 @@
+name: Check if a precompiled binary file exists in dependencies
+
+on:
+  pull_request:
+    paths:
+      python.lock
+
+jobs:
+  check-requirement:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Check if a precompiled binary file exists in dependencies
+        run: |
+            cat python.lock | grep -v '^//' | jq -r '.locked_resolves[].locked_requirements[]
+            | select(any(.artifacts[].url; endswith(".whl")) | not) | .project_name' >> no-exist-binary.txt
```

Comment:
* `no-exist-binary.txt` looks like too Konglish...
  Let's rename it to `packages-missing-precompiled-wheels`.

* What if the target package is a pure Python package that provides only the source distribution?